_posts/2018-01-18-From-Manual-to-Automatic.md

```diff
@@ -0,0 +1,111 @@
+
+Elegible product features that are candidates for test automation are usually those that were previously developed, the one that are considered stable or that have been already manually tested and with known results. On the other hand, unstable, temporary, incomplete features, or those whose automation is so complex, expensive, time-consuming and with high maintenance times are usually left out. This is due to the high initial costs of test automation which are only compensated and supported by successive regression test cycles being a medium for long product life cycles<sup>[3]</sup>.
+
+## Functional automation tools 
+One of the first questions that test automation raises refers to which languages, frameworks, libraries and tools can be used.
+To answer this, let´s describe some of them:
+
++ **RobotFramework** An open source test automation framework developed in Python, based on high-level keywords, which even users with rudimentary programming skills can use. Although this framework has been developed with acceptance tests in mind, the same is able a range of layers taht can go from testing Web GUIs, Databases, APIs and others, depending on which library is included in the framework. It is also possible to create custom keywords and libraries using your own framework or language like Java or Python. Another significant advantage is the quality of produced reports. Besides RobotFramework’s Builtin library, there are other relevant external libraries. The most commonly used are: 
+	
+	* RequestLibrary – Contains keywords to test Web APIs. Allows you to execute and evaluate HTTP requests, and to analyse messages exchanged with this protocol. 
+	* Selenium2Library – Interaction and evaluation of elements within an HTML page through interaction with DOM. Used essentially to test Web GUIs. 
+	* XMLLibrary – Allows for navigation, removal, editing and creation of elements within an XML structure. 
+	* JSONLibrary – Interaction with a JSON structure. 
+	* DatabaseLibrary – Interaction with databases.
+
++ **Python, Java and C#** - These are more complex programming languages that require more technical knowledge in test automation. They are also more powerful and flexible, allowing greater interaction with the machine, to produce lower level tests. 
+
+## Continuous Testing
+With *time to market* metrics getting increasingly audacious, one must adopt strategies that allow products to develop without compromises to their quality. Continuous Deployment (CD) is a short-cycle software production and deployment strategy that results in a shorter time to market<sup>[4]</sup>. It is only possible through Continuous Integration (CI), which translates in small, frequent<sup>[5]</sup> integration of new code within the existing one. 
```

Comment:
`one must adopt strategies that allow products to develop without compromises to their quality.` -> `one must adopt strategies that allow the development of products without compromising their quality.`

I also think `frequent<sup>[5]</sup> integration` should be in the plural.

Reply:
One or the other seems ok for me. I will change it to plural then.